tsconfig.json

```diff
@@ -17,6 +17,7 @@
       "dom"
     ],
     "strictPropertyInitialization": false,
+    "strict": true,
```

Comment:
Nitpick: Since it's enabled globally, we can drop explicit strict: true for individual libs (like qualtrics, tracking, etc...)

Btw. do I undestand correctly, that it's also applied for `storefrontapp-e2e-cypress` and it's all good? Or we need to apply strict:true there explicitly?

Reply:
I missed that and disabled that for cypress like for the rest of tests (unit)